README.md

```diff
@@ -20,7 +20,72 @@
 
 ### configuration and population
 
+The config file config.py got a lot of interesting setting:
 
+```python
+population_file_base="e2pieces.txt"
+NGEN = 2000
+mutate_inpd=0.01
+selection_ind_value_step=1
+elitism_percentage_start=10
+elitism_percentage_up=4
+gen_modulo_elitism=100
+```
+
+- population_file_base:
+- NGEN:
+
+### Workflows. Running Scripts
+
+  The workflow starts by running the script called main then the script render
+
+  ```bash
+  $ python main.py
+  [Error 183] Cannot create a file when that file already exists: './gen/'
+  Personal Path used for this Puzzle: gen/lumy_13-02-2017_13h.14m.46s/
+  No Solution Look at the logbook.
+  $ python render.py gen/lumy_13-02-2017_13h.14m.46s/
+  (it take few time before an ouput arrive because it's loading big binary file).
+  progression Bar 1 #################
+  progression Bar 2 ########________#
+  $ ./gen/generations_images_to_video.bash gen/lumy_13-02-2017_13h.14m.46s/ # Only linux
+  ```
+  > If you do not see any progressbar it might be because you're not in a real tty
+  (the package we use seems to work peferclty under all unix/linux but seems
+  still in dev for windows and others).
+
+  If you want benchmark our algorithm you can simply launch algo_benchmark.py as:
+
+  ```bash
+  $ python algo_benchmark.py
+  nb_parallel_executions 1
+  nb_executions_per_grid_size 4
+
+  launching benchmark...
+  [Error 183] Cannot create a file when that file already exists: './gen/'
+  ord() expected a character, but string of length 0 found

```

Comment:
We should delete every occurence of this error, as it is supposed to be a temporary bug